draft-ietf-mls-architecture.md

```diff
@@ -907,16 +907,22 @@ application could decide that a group administrator will be the only member to
 perform add and remove operations. On the other hand, in many settings such as
 open discussion forums, joining can be allowed for anyone.
 
-The MLS protocol can, in certain modes, exchange unencrypted group operation
-messages. This flexibility is to allow services to perform access control tasks
-on behalf of the group.
-
-While the Application messages will always be encrypted, having the handshake
-messages in plaintext has privacy consequences as someone could
-collect the signatures on the handshake messages and use them for tracking.
-
-> **RECOMMENDATION:** Prefer using encrypted group operation messages to avoid
-> privacy issues related to non-encrypted signatures.
+While MLS Application messages are always encrypted,
+MLS handshake messages can be sent encrypted (in an MLS
+PrivateMessage) or unencrypted (in an MLS PublicMessage). Applications
+may be designed such that intermediaries need to see handshake
+messages in order to enforce policy on which commits are allowed,
+and/or to provide MLS GroupInfo messages in a central location. If
```

Comment:
```suggestion
messages, for example to enforce policy on which commits are allowed,
or to provide MLS ratchet tree data in a central location. If
```

GroupInfo doesn't really have anything inherently private.  It's mainly the ratchet tree that is sensitive (in particular the credentials therein), and also the thing that is a pain to ship around.